components/style/stylist.rs

```diff
@@ -1504,7 +1517,12 @@ impl Stylist {
                             None,
                             &metrics,
                             CascadeFlags::empty(),
-                            self.quirks_mode)
+                            self.quirks_mode,
+                            self.registered_property_set
```

Comment:
nit: can use `self.registered_property_set()`, here and above.

Reply:
Great catch!